recipes/boost/all/conanfile.py

```diff
@@ -59,42 +87,67 @@ class BoostConan(ConanFile):
         "extra_b2_flags": "ANY",  # custom b2 flags
         "i18n_backend": ["iconv", "icu", None],
     }
-    options.update({"without_%s" % libname: [True, False] for libname in lib_list})
+    for _name in CONFIGURE_OPTIONS:
+        options["without_{}".format(_name)] = [True, False]
```

Comment:
```suggestion
    options.update({"without_{}".format(_name):[True, False] for _name in CONFIGURE_OPTIONS})
```

Reply:
Yes, this loop introduced a `_name` class variable for the `BoostConan` class.
I've updated it to your suggestion.